src/addrman.cpp

```diff
@@ -78,10 +78,10 @@ double CAddrInfo::GetChance(int64_t nNow) const
 }
 
 CAddrMan::CAddrMan(std::vector<bool> asmap, bool deterministic, int32_t consistency_check_ratio)
-    : m_asmap{std::move(asmap)}
-    , insecure_rand{deterministic}
+    : insecure_rand{deterministic}
     , nKey{deterministic ? uint256{1} : insecure_rand.rand256()}
     , m_consistency_check_ratio{consistency_check_ratio}
+    , m_asmap{std::move(asmap)}
```

Comment:
commit: 5840476714ffebb2599999c85a23b52ebcff6090  _[addrman] Make m_asmap private_

Is making the ordering of private members last in the initializer list a personal preference? I find it slightly odd that the initializer list no longer matches the order of the ctor args. Also you could have just introduced them in this order? =P

Reply:
The order here must be the same as the order in which the variables are declared in the class, otherwise we get a compiler warning.

Reply:
ah right, thanks

Reply:
The order could have been kept the same if it was moved to before `insecure_rand` in the header file